sourcegraph/sourcegraph
# Revert "[Wildcard] Migrate `data-tooltip` within `client/web` directo…

…ries (part 2 of 2) (#38118)"

This reverts commit abfcaec44d1eead64dd2f22fa9ce9f36ce0809fc.

## Reason for the revert
https://github.com/sourcegraph/sourcegraph/pull/38118 introduced a regression with "Show history" button no longer working, and perhaps some other things being broken too.

See slack thread: https://sourcegraph.slack.com/archives/C07KZF47K/p1657877422017449

## Test plan
1. Pull the branch
2. Check that "Show history" works
3. Check that tooltips still work too

## App preview:

- [Web](https://sg-web-revert-data-tooltip-migration.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-pwanaqrlvr.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
